test/integration/aaa_download_only_test.go

```diff
@@ -158,6 +158,18 @@ func TestDownloadOnly(t *testing.T) {
 				}
 			})
 
+			t.Run("minikube logs", func(t *testing.T) {
+				ctx, cancel := context.WithTimeout(context.Background(), Seconds(5))
+				defer cancel()
+				args := []string{"logs", "-p", profile}
+				if _, err := Run(t, exec.CommandContext(ctx, Target(), args...)); err != nil {
+					t.Errorf("minikube logs failed with error: %v", err)
```

Comment:
can ignore minikube logs error here...just care about the duration.

Reply:
Can't error or linting fails, I replaced with a `t.Logf` so it won't fail the test.